Windows/Deploy-Base/Start.ps1

```diff
@@ -19,7 +19,7 @@ if ($Env:AZP_WORK -and -not (Test-Path Env:AZP_WORK)) {
 	New-Item $Env:AZP_WORK -ItemType directory | Out-Null
 }
 
-New-Item "\azp\agent" -ItemType directory | Out-Null
+New-Item "\azp\agent" -ItemType directory -ErrorAction SilentlyContinue | Out-Null
```

Comment:
This isn't really needed but when the agent starts up, it has an error of azp\agent already exists due to the volume mount adding in azp\agent\_work\_tasks